### PR TITLE
fix: docker image failed to boot on arm64 node

### LINKED ISSUE #2115: Deploy Meilisearch v0.25.2 on K8s arm64 node failed
**Describe the bug**

Deploy Meilisearch v0.25.2 on K8s arm64 node failed.
Pod log:
```
<jemalloc>: Unsupported system page size
<jemalloc>: Unsupported system page size
memory allocation of 5 bytes failed
```

**To Reproduce**

1. Apply the yaml <https://github.com/meilisearch/meilisearch-kubernetes/blob/main/manifests/meilisearch.yaml>

**Expected behavior**
No error.

**Screenshots**
No.

**MeiliSearch version:**

v0.25.2

**Additional context**
OS: Linux xxx.xxx.xxx.xxx 4.18.0-305.10.2.el7.aarch64 #1 SMP Fri Jul 23 21:19:40 UTC 2021 aarch64 aarch64 aarch64 GNU/Linux
Architecture: arm64

### PR DESCRIPTION
**Pull request by Thearas**
# Pull Request

## What does this PR do?
Fixes #2115.

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

### CODE REVIEW

**Comment by Thearas** [comment]
I have tested the binary on my `arm64` machine, it seems work:
image

But I haven't tested the docker image, because it compiles too slow. I'll build it on my machine later when I have time.

File: Dockerfile [hunk 1]
```diff
@@ -17,6 +17,10 @@ COPY    meilisearch-http/Cargo.toml meilisearch-http/
 COPY    meilisearch-lib/Cargo.toml meilisearch-lib/
 
 ENV     RUSTFLAGS="-C target-feature=-crt-static"
+# Issue #2115
```
Comment (curquiza) [inline Dockerfile:20]:
```suggestion

```

File: Dockerfile [hunk 2]
```diff
@@ -21,7 +21,10 @@ ENV     RUSTFLAGS="-C target-feature=-crt-static"
 # Create dummy main.rs files for each workspace member to be able to compile all the dependencies
 RUN     find . -type d -name "meilisearch-*" | xargs -I{} sh -c 'mkdir {}/src; echo "fn main() { }" > {}/src/main.rs;'
 # Use `cargo build` instead of `cargo vendor` because we need to not only download but compile dependencies too
-RUN     $HOME/.cargo/bin/cargo build --release
+RUN     if [ "$TARGETPLATFORM" = "linux/arm64" ]; then \
+            export JEMALLOC_SYS_WITH_LG_PAGE=16; \
+        fi && \
+        $HOME/.cargo/bin/cargo build --release
 # Cleanup dummy main.rs files
 RUN     find . -path "*/src/main.rs" -delete
```
Comment (Thearas) [inline Dockerfile:30]: Hi @curquiza, I tested it successfully on arm64 machine. But compiling the image is just too slow 😂.
BTW, can we remove the `cargo build` above? It does not seem necessary.

Comment (curquiza) [reply Dockerfile:30]: Hum, I would rather remove this line in another PR to ensure we do not any mistake by introducing JEMALLOC_SYS_LG_PAGE

**Review by curquiza — APPROVED** [review]
Thanks @Thearas 

bors merge

**Pull request merged**